### ISSUE TITLE
Wav2vec Hyperparameter (mask)

### ISSUE THREAD

**Issue by sooftware** [pytorch/fairseq#2617]
README recommends creating a dataset from a wav that is about 10-30 seconds long to pre-train Wav2vec 2.0.  
But the length of my dataset is about 5.6 seconds on average.  
  
I think that I should adjust hyperparameters.  
I guess `--mask-length`, `--mask-prob` right?
  
Pre-training takes a lot of time and money. I'd like to make a careful decision, so I'd be so grateful if you could help me.  
  
And either opinion would be too much appreciated. Please give me some advice.

**Comment by sooftware** [comment]
Oh, and will the batch size have a big impact on the pre-train?
